.github/CONTRIBUTING.md

```diff
@@ -2,8 +2,8 @@
 
 -   You can join us on [AnitaB.org Open Source Zulip](https://anitab-org.zulipchat.com/). Each active repo has its own stream to direct questions to (for example #powerup or #portal). Open Source Programs stream is [#open-source-progs](https://anitab-org.zulipchat.com/#streams/237907/open-source-progs).
```

Comment:
@The-Keshav-Agarwal  just change the the link & the name here also for zulip
Link : https://anitab-org.zulipchat.com/#narrow/stream/237907-anitab-forms
Name of the Stream : [#anitab-forms]